File: PizzeriaS3Client/model/ingredient.php
<?php
require_once("objet.php");
class ingredient extends objet {

protected static string $classe = "Ingredient";
protected static string $identifiant = "IDIngredient";

protected int $IDIngredient ;
protected string $NomIngredient;
protected ?int $QtIngredientStock;

// constructeur 
public function __construct(int $IDIngredient = NULL, string $NomIngredient = NULL, int $QtIngredientStock = NULL) {
    if(!is_null($IDIngredient)){
        $this->IDIngredient = $IDIngredient;
        $this->NomIngredient = $NomIngredient;
        $this->QtIngredientStock = $QtIngredientStock;
    }



}

public function getNomIngredient() {
    return $this->NomIngredient;
}

public function getIDIngredient() {
    return $this->IDIngredient;
}

public function __toString() {
    return "$this->NomIngredient $this->QtIngredientStock";

}

}

?>

File: PizzaioloCuisineProjetS3/src/modeles/Ingredient.java
package modeles;
public class Ingredient {

	private int IDIngredient;
	private String NomIngredient;
	private int nbIngredients;
	
	public Ingredient (int IDIngredient,String NomIngredient, int nbIngredients) {
		this.IDIngredient = IDIngredient;
		this.NomIngredient = NomIngredient;
		this.nbIngredients = nbIngredients;
	}
	
	public int getIDIngredient() {
		return IDIngredient;
	}
	public void setIDIngredient(int iDIngredient) {
		IDIngredient = iDIngredient;
	}
	public String getNomIngredient() {
		return NomIngredient;
	}
	public void setNomIngredient(String nomIngredient) {
		NomIngredient = nomIngredient;
	}
	public int getNbIngredients() {
		return nbIngredients;
	}
	public void setNbIngredients(int nbIngredients) {
		this.nbIngredients = nbIngredients;
	}

}